src/OrbitCaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -348,25 +350,31 @@ std::vector<TimerInfo> GpuQueueSubmissionProcessor::ProcessGpuDebugMarkers(
       const GpuJob* matching_begin_job = FindMatchingGpuJob(
           begin_marker_thread_id, begin_marker_meta_info.pre_submission_cpu_timestamp(),
           begin_marker_post_submission_cpu_timestamp);
-      CHECK(matching_begin_job != nullptr);
-
-      // Convert the GPU time to CPU time, based on the CPU time of the HW execution begin and the
-      // GPU timestamp of the begin of the first command buffer. Note that we will assume that the
-      // first command buffer starts execution right away as an approximation.
-      marker_timer.set_start(completed_marker.begin_marker().gpu_timestamp_ns() +
-                             matching_begin_job->gpu_hardware_start_time_ns() -
-                             begin_submission_first_command_buffer->begin_gpu_timestamp_ns());
+
+      uint64_t begin_submission_time_ns = 0;
+      // We might have bad luck and captured the "begin" submission, but not the matching job.
```

Comment:
This reads as if this comment applies to the "if" case, but the "bad luck" part is actually handled in the "else" clause. Move this comment into the "else" block?

Reply:
Done.